src/main/java/org/broadinstitute/hellbender/utils/pileup/ReadPileup.java

```diff
@@ -255,6 +265,13 @@ public int getNumberOfElements(final Predicate<PileupElement> peFilter){
         return (int)getElementStream().filter(peFilter).count();
     }
 
+	/**
+     * Shortcut for get the number of deletions
+     */
+    public int getNumberOfDeletions() {
```

Comment:
why do we need this method? can you just inline it in the caller? If it's used just once let's either inline or keep it private.